### PR TITLE
Add Primary Nav Mobile Story

### PR DESCRIPTION
**Pull request by hariscs**
## Describe your changes
created NavPrimaryMobile story

## Screenshots - If Any (Optional)
image

## Link to issue

Add NavPrimaryMobile story #167 

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [x] I ran the app and tested it locally to verify that it works as expected.
- [x] I have checked my code with an automatic accessibility tool such as Axe Dev Tools or Wave 
      and it shows no errors.
## Additional Information (Optional)
Any additional information that you want to give us.

<a href="https://gitpod.io/#https://github.com/AccessibleForAll/AccessibleWebDev/pull/173"></a>

### CODE REVIEW

**Comment by EmmaDawsonDev** [comment]
Thanks so much for your PR! The review is on hold until myself and @ctoffanin have checked something with the storybook setup tomorrow, then we’ll get back to reviewing this 😊

File: stories/NavPrimaryMobile.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+} as ComponentMeta<typeof NavPrimaryMobile>
+
+const Template: ComponentStory<typeof NavPrimaryMobile> = (args) => (
+	<NavPrimaryMobile {...args} />
+)
+
+export const MobilePrimaryNav = Template.bind({})
+
+MobilePrimaryNav.args = {
+	activeNavLink: "home",
```
Comment (EmmaDawsonDev) [inline stories/NavPrimaryMobile.stories.tsx:17]:
```suggestion
	activeNavLink: "/home",
```

I think I may have written this wrong in the instructions. Can you add a / and see if the active class starts showing up on home?

Comment (hariscs) [reply stories/NavPrimaryMobile.stories.tsx:17]: sure, lemme try

Comment (hariscs) [reply stories/NavPrimaryMobile.stories.tsx:17]: yes, it worked, pushing the new changes

**Review by EmmaDawsonDev — APPROVED** [review]

**Comment by EmmaDawsonDev** [comment]
Thanks for contributing!

**Pull request merged**